ui-core/src/styles/inputs/statusMessage.css

```
@@ -31,3 +31,60 @@
     }
   }
 }
+
+.status-message-wrapper-tooltip {
+  position: absolute;
+  display: flex;
+  align-items: center;
+  width: 320px;
+  top: 74px;
+  gap: 12px;
+  border-radius: 8px;
+  box-shadow:
+    0 10px 20px 0 rgba(0, 0, 0, 0.2),
+    0 1px 0 rgba(255, 255, 255, 0.6) inset;
+  padding: 6px 12px 10px;
+  min-height: 64px;
+  z-index: 10;
+
+  &.small {
+    top: 58px;
+  }
+
+  &.error {
+    background-color: theme('colors.error.5');
+  }
+
+  &.info {
+    background-color: theme('colors.info.5');
+  }
+
+  &.warning {
+    background-color: theme('colors.warning.5');
+  }
+
```

You can also add success (same for `.status-message` `color.60`)
```suggestion
  &.success {
    background-color: theme('colors.success.5');
  }
```